Frontend/android-native/app/src/main/java/in/juspay/mobility/ResourceHandler.java

```diff
@@ -0,0 +1,91 @@
+package in.juspay.mobility;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import in.juspay.hyper.constants.LogCategory;
+import in.juspay.hyper.constants.LogSubCategory;
+import in.juspay.hypersdk.core.JuspayServices;
+
+public class ResourceHandler {
+
+    @NonNull
+    private final Context context;
+    @NonNull
+    private final JuspayServices juspayServices;
+
+    private final String LOG_TAG = this.getClass().getSimpleName();
+
+    public ResourceHandler(@NonNull Context context) {
+        this.context = context;
+        this.juspayServices = new JuspayServices(context,null);
+    }
+
+    protected String getFromAssets (String fileName) {
+        return juspayServices.getFileProviderService().readFromFile(context,fileName);
+    }
+
+    @SuppressLint("DiscouragedApi")
+    protected boolean isResourcePresent (String type, String fileName) {
+        return (context.getResources().getIdentifier(fileName, type, context.getPackageName()) != 0);
+    }
+
+
+    /**
+     * Make sure the file exist before calling this function by isResourcePresent
+     * @param fileName Resource file name
+     * @return raw file in String format
+     */
+    @SuppressLint("DiscouragedApi")
```

Comment:
there's an alternative for this? `DiscouragedApi's` are slow and not suggested to implement

Reply:
Alternate is we have to put the file in local and use. It won't support dynamic assets fetch.